CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.1.0
```

Comment:
Since its a breaking change, lets make this version 3.0.0

Reply:
will do.